app/commands.py

```diff
@@ -248,3 +248,23 @@ def run(self):
         db.session.commit()
 
         print("Populated email reply to adderesses for {}".format(result.rowcount))
+
+
+class PopulateServiceLetterContact(Command):
```

Comment:
Should there be a test for this?

Reply:
This command will only ever be run once in each environment - we'll test it by checking the database before and after running it to make sure it has worked as expected.